### PR TITLE
IsPathFullyQualified() Tests, public API

### PR DESCRIPTION
**Pull request by niustephanie**
Added tests for Path.IsPathFullyQualified() for Windows and Unix, made IsPathFullyQualified() public.

### CODE REVIEW

File: src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs [hunk 1]
```diff
@@ -1178,6 +1178,7 @@ public static partial class Path
         public static string GetTempPath() { throw null; }
         public static bool HasExtension(string path) { throw null; }
         public static bool IsPathRooted(string path) { throw null; }
+        public static bool IsPathFullyQualified(string path) { throw null; }
```
Comment (ViktorHofer) [inline src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs]: the title of the PR suggests that you are only adding tests but here you are adding a public API. Intentional?

Comment (JeremyKuhne) [inline src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs]: Yes, this is intentional. @niustephanie Can you update the description that this also exposes the new API in the ref?

File: src/System.Runtime.Extensions/tests/System/IO/PathTests.cs [hunk 2]
```diff
@@ -311,6 +311,81 @@ public static void IsPathRooted_Windows_Invalid(string value)
...
+            Assert.Throws<ArgumentNullException>(() => Path.IsPathFullyQualified(null));
+        }
+
+        [Fact]
+        public static void IsPathFullyQualified_Empty()
+        {
+            Assert.False(Path.IsPathFullyQualified(""));
+        }
+
+        [PlatformSpecific(TestPlatforms.Windows)]
```
Comment (JeremyKuhne) [inline src/System.Runtime.Extensions/tests/System/IO/PathTests.cs]: This needs conditionalized to not run on desktop (needs [SkipOnTargetFramework(...)]

Comment (JeremyKuhne) [inline src/System.Runtime.Extensions/tests/System/IO/PathTests.cs]: Actually I think we need to put this in a separate .netcoreapp.cs file.  Not sure if our UAP runtime has picked up the new Path bits either, so might need conditionalized for that as well for now.

File: src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs (hunk 1 above)
Comment (niustephanie) [inline src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs]: Yep!

**Comment by niustephanie** [comment]
@dotnet-bot test NETFX x86 Release Build failed

**Comment by niustephanie** [comment]
@dotnet-bot test UWP NETNative x86 Release Build failed

**Comment by niustephanie** [comment]
@dotnet-bot test UWP CoreCLR x64 Debug Build failed

**Pull request merged**